site/src/examples/static-list/AdditionalLabel.tsx

```diff
@@ -0,0 +1,27 @@
+import { Label, StackLayout, Text } from "@salt-ds/core";
+import { StaticList, StaticListItem } from "@salt-ds/lab";
+import type { ReactElement } from "react";
+
+const ListItem = () => (
+  <StaticListItem>
+    <StackLayout
+      gap={0.5}
+      style={{ padding: "var(--salt-spacing-75) 0", width: "100%" }}
```

Comment:
The `padding` overrides stood out for me. Are we having a good default for the static list item?